### PR TITLE
Set up metadata and options for setup function in setup.cfg

### PR DESCRIPTION
**Pull request by naveen-ram**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -7,4 +7,29 @@ default=db2+pyodbc://db2inst2:db2inst2@/test
...
+    Topic :: Databases :: Front-end, middle-tier
+platforms=All
+
+[options]
+zip_safe = False
+include_package_data = True
+install_requires =
+    sqlalchemy>=1.3
+    pyodbc>=4.0
+packages= sqlalchemy_ibmi
```
Comment (kadler) [inline setup.cfg]:
```suggestion
packages = sqlalchemy_ibmi
```

File: setup.cfg [hunk 2]
```diff
@@ -7,4 +7,29 @@ default=db2+pyodbc://db2inst2:db2inst2@/test
...
+author = IBM
+license = Apache License 2.0
+keywords = sqlalchemy, database, ibm, ibmi, db2
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache License 2.0
+    Operating System :: OS Independent
+    Topic :: Databases :: Front-end, middle-tier
+platforms=All
```
Comment (kadler) [inline setup.cfg]:
```suggestion
platforms = All
```

File: setup.cfg [hunk 3]
```diff
@@ -7,4 +7,29 @@ default=db2+pyodbc://db2inst2:db2inst2@/test
 sqlite=sqlite:///:memory:
 
 [metadata]
+name = sqlalchemy-ibmi
 version = attr: sqlalchemy_ibmi.__version__
+description = SQLAlchemy support for Db2 on IBM i
+long_description = file: README.md
```
Comment (kadler) [inline setup.cfg]: Do we need to set `long_description_content_type` or does it automatically figure it out from the file extension?

Comment (naveen-ram) [reply setup.cfg]: Have you seen this elsewhere? I believe the above is the best way to implement the long description. Followed example I saw on https://setuptools.readthedocs.io/en/latest/setuptools.html

Comment (kadler) [reply setup.cfg]: That example uses a file in ReStructured text, which is the default. `long_description_content_type` was added to allow for markdown.

See https://packaging.python.org/guides/making-a-pypi-friendly-readme/ and https://packaging.python.org/specifications/core-metadata/#description-content-type-optional

> If a Description-Content-Type is not specified, then applications should attempt to render it as text/x-rst; charset=UTF-8 and fall back to text/plain if it is not valid rst.

My question was whether setuptools has any magic to determine that from the file name or if it still defaults to rst.

Comment (naveen-ram) [reply setup.cfg]: I think that it does need to be set, I think I misread your initial comment. Will add that to the config file.

**Pull request merged**